list-sensors.cpp

```diff
@@ -354,7 +347,14 @@ int main(int argc, char* argv[])
     using Objects = std::map<Path, ObjectsMap, cmp_sensors_name>;
 
     Objects objects;
-    reply.read(objects);
+    try
+    {
+        systemBus.call(method).read(objects);
+    }
+    catch (const sdbusplus::exception::SdBusError& ex)
+    {
+        fprintf(stderr, "Error: %s\n", ex.what());
```

Comment:
Don't we need to abort here with `EXIT_FAILURE`? This code will print an error, but will fall through to line 367 and return a success code.

Reply:
Fixed